dialogue-core/src/main/java/com/palantir/dialogue/core/CautiousIncreaseAggressiveDecreaseConcurrencyLimiter.java

```diff
@@ -137,7 +142,7 @@ void success() {
         SUCCESS() {
             @Override
             public double applyAsDouble(double originalLimit, double inFlightSnapshot) {
-                if (inFlightSnapshot >= originalLimit * BACKOFF_RATIO) {
+                if (inFlightSnapshot >= Math.floor(originalLimit) * BACKOFF_RATIO) {
```

Comment:
nice

Reply:
Do we want to match the `DROPPED` case and wrap the result instead?
```suggestion
                if (inFlightSnapshot >= Math.floor(originalLimit * BACKOFF_RATIO)) {
```

Reply:
shouldn't make much difference either way

Reply:
I've updated it just for consistency - I think the difference this makes is we'd increase a little bit more eagerly when the limit is low, but overall yeah doesn't seem that big.